apps/web/components/eventtype/RecurringEventController.tsx

```diff
@@ -47,6 +47,11 @@ export default function RecurringEventController({
           <Alert severity="warning" title={t("warning_payment_recurring_event")} />
         ) : (
           <>
+            <Alert
+              className="mb-4"
+              severity="warning"
+              title="Experimental: Recurring Events are currently experimental and causes some issues sometimes when checking for availability. We are working on fixing this."
+            />
```

Comment:
we are not translating this, because we plan to remove this warning and don't want extra i18n clutter for a temporarily warning